loone_data_prep/GEOGLOWS_LOONE_DATA_PREP.py

```diff
@@ -61,53 +62,67 @@ def main(input_dir: str, output_dir: str, ensemble_number: str) -> None:  # , hi
     LO_Stg_Sto_SA_df["SA_acres"] = LO_SA  # acres
 
     # Using geoglows data for S65_total, only data from S65E_S (none from S65EX1_S)
-    S65_total = pd.read_csv(f"{input_dir}/S65E_S_FLOW_cmd_geoglows.csv")
+    S65_total = pd.read_csv(f"{input_dir}/750072741_INFLOW_cmd_geoglows.csv")

```

Comment:
Please retain the original file names based on stations instead of geoglows river IDs. I suggest you create a dictionary that maps the stations to the river ids and use that throughout the code to access the geoglows data. This applies to all the changes here.